test/system/src/test/java/io/pravega/test/system/ReadWriteAndAutoScaleWithFailoverTest.java

```diff
@@ -170,26 +167,6 @@ public void readWriteAndAutoScaleWithFailoverTest() throws Exception {
         validateResults(readerGroupManager, readerGroupName);
 
         cleanUp(scope, AUTO_SCALE_STREAM); //cleanup if validation is successful.
-    }
-
-    private void waitForScaling() throws InterruptedException, ExecutionException {
-        for (int waitCounter = 0; waitCounter < 2; waitCounter++) {
-            StreamSegments streamSegments = controller.getCurrentSegments(scope, AUTO_SCALE_STREAM).get();
-            testState.currentNumOfSegments.set(streamSegments.getSegments().size());
-            if (testState.currentNumOfSegments.get() == 2) {
-                log.info("The current number of segments is equal to 2, ScaleOperation did not happen");
-                //Scaling operation did not happen, wait
-                Exceptions.handleInterrupted(() -> Thread.sleep(60000));
-                throw new AbstractFailoverTests.ScaleOperationNotDoneException();
-            }
-            if (testState.currentNumOfSegments.get() > 2) {
-                //scale operation successful.
-                log.info("Current Number of segments is {}", testState.currentNumOfSegments.get());
-                break;
-            }
-        }
-        if (testState.currentNumOfSegments.get() == 2) {
-            Assert.fail("Current number of Segments reduced to less than 2. Failure of test");
-        }
+        log.info("Test {} succeeds ", "ReadWriteAndAutoScaleWithFailover");
```

Comment:
nit: we could remove parameters since it is a hard coded statement, or remove the log statement itself. Similar pattern in other tests.

Reply:
done.